app/gateways/websocket_gateway.go

```diff
@@ -68,8 +82,21 @@ func (wg *WorkspaceGateway) OnWorkspaceStartEvent(s socketio.Conn, data map[stri
 	ctx := s.Context().(map[string]interface{})
 	ctx["project_id"] = projectIDStr
 	s.Emit("workspace-started", fmt.Sprintf("Workspace started for project: %v", projectID))
+	
+	channel := fmt.Sprintf("project-notifications-%d", projectID)
+	pubsub, err := wg.projectNotificationService.ReceiveNotification(func(msg string) {
+		s.Emit("projectNotification", msg)
+		wg.logger.Info("_____message sent to frontend ",zap.Any("with connection id____ ", s.ID()))
```

Comment:
this has to be connection id what is `with connection id____ `, shouldn't this be debug log?